ios/MaasCore/Sources/MaasCore/Api/ApiLogger.swift

```diff
@@ -1,20 +1,21 @@
 import os.log
-
-class ApiLogger : CoreBinary.Logger {
-
-    private init() {}
-    static let `default` = ApiLogger()
-    
-    var apiLogLevel: ApiLogLevel { .all }
-    
-    func log(message: String) {
-        os_log("%@", log: .MaasComponentsApi, type: .debug, message)
-    }
-}
-
-extension OSLog {
-    
-    private static var subsystem = Bundle.main.bundleIdentifier!
-    
-    static let MaasComponentsApi = OSLog(subsystem: subsystem, category: "MaasComponents")
-}
+import CoreBinary
+//
+//class ApiLogger : CoreBinary.Logger {
```

Comment:
This could be deleted as it is an example, ant there is one in Login sample

Reply:
Sure i'll delete this. 🙇